pages/science/reactors/customreactor.rst

```diff
@@ -0,0 +1,45 @@
+.. title: Custom Reactors in Cantera
+.. has_math: true
+
+.. jumbotron::
+
+   .. raw:: html
+
+      <h1 class="display-3">Custom Reactors in Cantera</h1>
+
+   .. class:: lead
+
+      This guide defines and links to an example of a Custom Reactor.
+
+Custom Reactors
+***************
+
+In some cases, Cantera's solver is insufficient to describe 
+a certain configuration. In this situation, Cantera can 
+still be used to provide chemical and thermodynamic computations, 
+but external ODE solvers can be applied. The user may entirely bypass 
+the provided governing equations, shown in the custom.py example 
+linked below. This demonstrates an approach 
+for solving problems where Cantera's reactor network model cannot 
+be configured to describe the system in question, as in a `Delegated
+Reactor </science/reactors/delegatedreactor.html>`__.
+
+The process to bypass Cantera's existing governing equations can be
+described in 7 steps:
+
+#. Define your new class
+   
+   a. Create _init_ function
+   b. Specify the new custom ODEs
+
+#. Create a new gas object and set initial conditions
+#. Integrate custom equations over a single timestep
+#. Set new gas state to the values obtained by the supplied ODE solver
+#. Save the gas state
+#. Repeat integration over all timesteps (steps 3-5)
+#. The final state vector contains all gas properties obtained from Cantera for each timestep
```

Comment:
This doesn't really make sense as a description of how the ODEs are solved. I think we need to iterate a bit on this to make it more useful and technically correct.